astro/src/content/docs/get-started/use-cases/app-suite.mdx

```diff
@@ -0,0 +1,124 @@
+
+To do so, update the `OAuth Authorize` template. This is the login page. 
+
+Here's a sample FreeMarker block to replace the default `rememberDevice` checkbox.
+
+```
+[@hidden name="rememberDevice" value="true"/]
+```
+
+The duration of this SSO session is controlled in the tenant settings.
+
+### Forward Unauthenticated Users
+
+Whenever a user accesses an application and is not logged in, forward them to the login URL for FusionAuth.
+
+Since your applications are already delegating authentication to FusionAuth, use the same logic to create the login URL.
+
+Here's a diagram of a user switching between the clownwear ecommerce store and the forum.
+
+<ApplicationSwitchDiagram alt="An example of an application switch." />
+
+If a user is browsing the ecommerce store, then clicks on the forum link, they will transparently be logged in to the forum application.
+
+Their client will get a token associated with the forum application, including the roles the user has for that application and claims such as the audience claim (`aud`) set correctly.
+
+## Expected Outcome
+
+Users switch between your applications transparently and securely. 
+
+## Edge Cases
+
+SSO login works across all apps in a tenant, including those the user isn't registered for. After bouncing to FusionAuth, the user will be authenticated, but the `applicationId` and `roles` claims will be missing from the access token. Your applications should check these claims when determining whether a user is authorized for an application. [Read more about authentication and authorization](/docs/get-started/core-concepts/authentication-authorization).
+
+When your user logs out, destroy the SSO session as each application session. You can do that using API calls or with FusionAuth's Front-Channel logout. [Read more about logout](/docs/lifecycle/authenticate-users/logout-session-management).
+
+Any applications that have self-service registration enabled will automatically register users for with default roles when the user is bounced to that application. 
+
+If an application only wants to allow logins from a certain identity provider or impose other conditions, you'll need extra integration work. For example, imagine there's a time tracking app for your clownwear ecommerce store that employees log in to using Google Workspace federation. You don't want customers to be able to view this application. One way you can prevent this with a [login validation lambda](/docs/extend/code/lambdas/login-validation).
+
+This use case only works with FusionAuth hosted login pages. If you use the Login API, you need to build your SSO. See this [GitHub issue for more details](https://github.com/fusionauth/fusionauth-issues/issues/1515).
```

Comment:
```suggestion
This use case only works with FusionAuth hosted login pages. If you use the Login API, you need to build your SSO. See this [GitHub issue for more details](https://github.com/fusionauth/fusionauth-issues/issues/1515).
```

I am not sure why, but I don't love linking this complex issue here.  This SSO creation is complex and I don't think that this GH issue explains the problem space fully.  We might solve SSO a different way / integrate existing SSO with the APIs in a different way in the future.  Dealers choice I suppose 😄

Reply:
Fair enough. Removed the link.